libexec/scoop-import.ps1

```diff
@@ -59,6 +63,14 @@ foreach ($item in $import.apps) {
 
     & "$PSScriptRoot\scoop-install.ps1" $app @instArgs
 
+    if ($update) {
+        & "$PSScriptRoot\scoop-update.ps1" $app$global
```

Comment:
Can this passing args correctly?

Reply:
No, $global does not exist. You are right. Fix is on the way.